sqlalchemy_continuum/manager.py

```diff
@@ -186,20 +186,22 @@ def is_excluded_property(self, model, key):
             return False
         return key in self.option(model, 'exclude')
 
-    def option(self, model, name):
+    def option(self, model_or_table, name):
         """
         Returns the option value for given model. If the option is not found
         from given model falls back to default values of this manager object.
         If the option is not found from this manager object either this method
         throws a KeyError.
 
-        :param model: SQLAlchemy declarative object
+        :param model_or_table: SQLAlchemy declarative object
         :param name: name of the versioning option
         """
-        if not hasattr(model, '__versioned__'):
-            raise TypeError('Model %r is not versioned.' % model)
+        if not hasattr(model_or_table, '__versioned__'):
+            if isinstance(model_or_table, sa.Table): # with #299 table could also have versioning_manager available
+                return self.options[name]
+            raise TypeError('Model %r is not versioned.' % model_or_table)
         try:
-            return model.__versioned__[name]
+            return model_or_table.__versioned__[name]
```

Comment:
Minor - this would be clearer to read with:
```py
if isinstance(model_or_table, sa.Table):
    return self.options[name]

try:
    return model_or_table.__versioned__[name]
except AttributeError:  # __versioned__ not found
    raise TypeError('Model %r is not versioned.' % model_or_table)
except KeyError:  # name is not overridden in model.__versioned__
    return self.options[name]
```

Reply:
Hi @AbdealiJK ,

Tried to address this added testcase as well added a model attr to table in table_builder for this